src/components/layout/PrimaryLayout/PrimaryLayout.js

```diff
@@ -19,7 +21,24 @@ export function PrimaryLayout ({
   showSideMenu = true,
   showAlphaFlag = true
 }) {
-  const { data: { hposSettings: settings = {} } = {} } = useQuery(HposSettingsQuery)
+  const { setIsConnected, isConnected } = useConnectionContext()
+
+  const onError = ({ graphQLErrors }) => {
+    const { isHposConnectionActive } = graphQLErrors
+    setIsConnected(isHposConnectionActive)
+  }
+
+  const { error, data: { hposSettings: settings = {} } = {} } = useQuery(HposSettingsQuery, { pollInterval: 10000, onError, notifyOnNetworkStatusChange: true, ssr: false })
```

Comment:
what's the purpose of the `ssr` flag here?

Reply:
This ensures that the login page still appears when the server is down and the connection isn't made. (Since we included the `notifyOnNetworkStatusChange` option, this catches and change in connection to the server, we will need this new flag in place.)